ntp-proto/src/packet/extensionfields.rs

```diff
@@ -549,10 +562,6 @@ impl<'a> RawEncryptedField<'a> {
         let nonce_length = u16::from_be_bytes([b0, b1]) as usize;
         let ciphertext_length = u16::from_be_bytes([b2, b3]) as usize;
 
-        if nonce_length != 16 {
-            return Err(IncorrectLength);
-        }
-
```

Comment:
this constraint is now no longer required, and an implementation detail of the ciphers that we provide today.